frontend/src/pages/spotnet/documentation/Documentation.jsx

```diff
@@ -12,9 +13,10 @@ const Documentation = () => {
 
 
   const tableOfContents = [
```

Comment:
<img width="372" alt="image" src="https://github.com/user-attachments/assets/57a86e8f-a9a2-46cc-ab51-ee9ee9e20df7" />

- Icorrect title styles
- no icons near the items and seems like sub-items do not work correctly
- when clicking `Overview` I can't select any other item anymore
- when I click getting started, all items below is highlighted, also those items can't be directly selected